README.template.md

```diff
@@ -70,6 +70,21 @@ echo 'export PATH="$PATH:$HOME/.bbin/bin"' >> ~/.zshrc && exec /bin/zsh
 echo 'export PATH="$PATH:$HOME/.bbin/bin"' >> ~/.bashrc && exec /bin/bash
 ```
 
+### Windows (Manual)
+
+**1. Install `bbin` CLI (and batch file, because Windows):**
+```shell
+> set BBIN_DEST=%HOMEDRIVE%%HOMEPATH%\.bbin\bin
+> mkdir %BBIN_DEST% && curl -o %BBIN_DEST%\bbin -L https://raw.githubusercontent.com/babashka/bbin/v{{version}}/bbin && curl -o %BBIN_DEST%\bbin.bat -L https://raw.githubusercontent.com/babashka/bbin/v{{version}}/bbin.bat
+> set BBIN_DEST=
+```
+
+**2. Add `%HOMEDRIVE%%HOMEPATH%\.bbin\bin` to `Path` environment variable**
+
+Because `Path` is often a system-level and user-level variable, setting from the command line can be messy; it's probably easiest to add it in the Environment Variables settings window.
+
+Also, remember to check line endings.
```

Comment:
Can you clarify what this means (in a GitHub comment)?

Reply:
There's a system Path variable (which would be vaguely analogous to the system-wide path that gets set in `/etc/profile` or `/etc/login.defs`), and a user-level Path variable. In a given windows 'seesion', the user-level Path is appended to the system-level Path, so you get roughly the same effect as a shell session where e.g. `.bashrc` appends to the $PATH that gets initialized by system-level things. 

However, in a shell session, tweaking PATH affects that session. This can be done in windows with the `set` or `path` commands, but to get that setting to live on, you need a login script or something similar (analogous to .bashrc) that tweaks Path every startup (which afaik is not a common thing, and seems like overkill to add a directory to Path). 

An alternative is using `setx` (which is like set, but it persists beyond the session), but this comes with headaches - best case it prepends the system path to your user path; worst case it truncates the Path because of command length limitations and you potentially lose some of your user-level Path (ask me how I know).

Reply:
That's helpful, thanks for the context.

Can you also clarify the part about line endings? What specifically should the user be checking for (as suggested by the original README)?

Reply:
My worry was that by curling the files from github (especially the batch file), the file might come down with Unix line endings. Having played around with it further, it looks like Windows generally handles this (e.g. a batch file with Unix line endings seems to be functionally equivalent to the same file with Windows line endings).